pkg/usagetracker/tracker_store.go

```diff
@@ -300,16 +341,54 @@ func (shard *tenantShard) trackSeries(series []uint64, now minutes, totalTenantS
 				created = append(created, s)
 				totalTenantSeries.Inc()
 			}
-			pending = pending[1:]
+			missing = missing[1:]
 		}
 		shard.Unlock()
 	}
 
-	// Whatever is pending is rejected (if any)
-	rejected = append(rejected, pending...)
+	// Whatever is still missing is rejected (if any)
+	rejected = append(rejected, missing...)
 	return created, rejected
 }
 
+// processCreatedSeriesEvent creates the series coming from an event from Kafka,
+// i.e. series that were created by a different instance.
+// This does not check the limits, as we prioritize staying consistent across replicas over enforcing limits.
+// Timestamp is not updated if series exists already.
```

Comment:
Shouldn't be updated with the highest value between the two? I guess it shouldn't happen, but never know.

Reply:
It could happen but we don't really care as we'll update when we process the snapshot later.